posts/2021/jupyter-based-diagnostics-overview.ipynb

```diff
@@ -0,0 +1,434 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reimagining Diagnostics Through the Use of the Jupyter Ecosystem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview of \"Traditional\" Diagnostics Packages\n",
```

Comment:
I don't think this subtitle is necessary, the paragraph in this subsection works well with the "Reimagining Diagnostics Through the Use of the Jupyter Ecosystem" title

Reply:
I took out that heading!